### PR TITLE
Add jquery.sumoselect@3.0.2 w/ git auto-update

### PR DESCRIPTION
**Pull request by clairetsai818**
Pull request for issue: #8169 
Related issue(s): # #

@cdnjs/intern3 Please help me review this PR. Thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/HemantNegi/jquery.sumoselect
 * License and its reference: MIT ,[ref](https://github.com/HemantNegi/jquery.sumoselect#LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 37
   - Count of stars: 287
   - Count of forks: 150

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery.sumoselect/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "author": "HemantNegi",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/HemantNegi/jquery.sumoselect.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.sumoselect.!(*json)"
```
Comment (hare1039) [inline ajax/libs/jquery.sumoselect/package.json]: I think we also need `sumoselect.css`

File: ajax/libs/jquery.sumoselect/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "description": "A jQuery plugin that progressively enhances an HTML Select Box into a Single/Multiple option dropdown list",
+  "filename": "jquery.sumoselect.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/HemantNegi/jquery.sumoselect.git"
+  },
+  "keywords": [
+    "multiselect",
+    "dropdown"
+  ],
```
Comment (hare1039) [inline ajax/libs/jquery.sumoselect/package.json]: I think `https://hemantnegi.github.io/jquery.sumoselect/sumoselect_demo.html` can be set as `homepage`.
What do you think?

Comment (extend1994) [inline ajax/libs/jquery.sumoselect/package.json]: Agree.

**Comment by clairetsai818** [comment]
I added `homepage` info and included `sumoselect.css`.
@cdnjs/intern3 please help me review this PR again, thanks!

File: ajax/libs/jquery.sumoselect/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "license": "MIT",
+  "homepage": "https://hemantnegi.github.io/jquery.sumoselect/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/HemantNegi/jquery.sumoselect.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.sumoselect.!(*json)"
```
Comment (extend1994) [inline ajax/libs/jquery.sumoselect/package.json]: You should also modify `package.json` to include `sumoselect.css` in other versions.

**Comment by clairetsai818** [comment]
@extend1994 I revised file map and minified `sumoselect.css`. Please help me review this again, thanks!

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me review this, thanks!

**Comment by PeterDaveHello** [comment]
@pvnr0082t please take a look, thanks.

File: ajax/libs/jquery.sumoselect/package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery.sumoselect",
+  "version": "3.0.2",
+  "description": "A jQuery plugin that progressively enhances an HTML Select Box into a Single/Multiple option dropdown list",
```
Comment (pvnr0082t) [inline ajax/libs/jquery.sumoselect/package.json]: @clairetsai818 Why not use `description` and `keywords` from upstream?
ref: **description**: https://github.com/HemantNegi/jquery.sumoselect/blob/v3.0.2/jquery.sumoselect.jquery.json#L5
**keywords**: https://github.com/HemantNegi/jquery.sumoselect/blob/v3.0.2/jquery.sumoselect.jquery.json#L6-L22

Comment (clairetsai818) [inline ajax/libs/jquery.sumoselect/package.json]: @pvnr0082t I used `description` and `keywords` in `package.json`, and I revised them. Please help me review this, thanks!

**Comment by PeterDaveHello** [comment]
I wonder if the css file is part of the release or is it just an example? As you can see the version release/bump doesn't include it, can you confirm that with the author?

**Comment by clairetsai818** [comment]
@PeterDaveHello There're no response from the contributors of this library, should we keep waiting for a while?
issue: https://github.com/HemantNegi/jquery.sumoselect/issues/176

**Comment by PeterDaveHello** [comment]
Okay, let's just host it, please rebase your branch with the latest master branch, and ask for review again, thanks.

**Comment by clairetsai818** [comment]
@PeterDaveHello I rebase this branch, please help me review this, thanks!

**Pull request merged**